core/src/main/java/org/infinispan/marshall/protostream/impl/SerializationContextRegistryImpl.java

```diff
@@ -57,18 +57,14 @@ public void start() {
 
       String messageName = PersistenceContextInitializer.getFqTypeName(MarshallableUserObject.class);
       BaseMarshaller userObjectMarshaller = new MarshallableUserObject.Marshaller(messageName, userMarshaller.wired());
-      update(GLOBAL, ctx -> ctx.addContextInitializer(commonTypesSchema)
-            .addContextInitializer(commonContainerTypesSchema)
-            .addContextInitializer(new PersistenceContextInitializerImpl())
+      update(GLOBAL, ctx -> ctx.addContextInitializer(new PersistenceContextInitializerImpl())
```

Comment:
Shouldn't this have been `new GlobalContextInitializerImpl`?

Reply:
No, a `GlobalContextInitializerImpl` does not exist in core. Currently there is only `org.infinispan.commons.GlobalContextInitializerImpl()`